### ISSUE TITLE
Packager does not start from extension

### ISSUE THREAD

**Issue by SounD120** [microsoft/vscode-react-native#1453]
in my case this fix doesn't work. Anything else I could try? updated everything, also xdl so now I don't have anything. This doesn't happen necessarily debugging, but as soon as I open vs code, wait for extension to load, and then "start packager" on the bottom bar.

```
[Info] Starting Packager

env: node: No such file or directory
[Error] Failed to start the React Native packager (error code 106)

Stack: Error: Failed to start the React Native packager (error code 106): Could not start the packager.
    at Function.getWrappedError (/Users/superandrew/.vscode/extensions/msjsdiag.vscode-react-native-1.1.0/dist/rn-extension.js:1:152512)
    at Function.wrapError (/Users/superandrew/.vscode/extensions/msjsdiag.vscode-react-native-1.1.0/dist/rn-extension.js:1:7049)
    at /Users/superandrew/.vscode/extensions/msjsdiag.vscode-react-native-1.1.0/dist/rn-extension.js:1:202075
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

_Originally posted by @superandrew in https://github.com/microsoft/vscode-react-native/issues/1440#issuecomment-721569925_

**Comment by superandrew** [comment]
@SounD120 Is there any way I could try "resetting" my VS Code situation to check if this still persists? The consistent fix seems to be to open developer tools and hit CMD+R

**Comment by SounD120** [comment]
@superandrew The fastest way to reset this that I can propose is to temporary comment all NVM entries in your `.zshrc` file and try to install Node.js via brew command and then add the path to this Node.js installation to `.zshrc`.  This might help to understand if the problem is whether in NVM Node.js configuration or not.

**Comment by superandrew** [comment]
what if I comment nvm and point in `.zhsrc` to a nvm installation? It should be the same thing, right? I'm a bit unconmfortable messing with node because I depend on it on so many tasks

**Comment by SounD120** [comment]
Yes, I think that should work the same way.

**Comment by superandrew** [comment]
hi @SounD120 even adding a valid node path doesn't seem to change. If I have the Jest extension loaded it prints many times the same error, so I guess it is not something that is pertinent to this extension but to VS Code or my installation. I will try to break the setup/reinstall and let you know my findings

**Comment by JiglioNero** [comment]
Hi @superandrew it looks like the problem is somewhere in the specific configuration on your machine and we could not reproduce this problem, we will leave this issue open, please let us know if you find any clues.

**Comment by SounD120** [comment]
Hi @superandrew . Do you have any `react-native-tools` settings in your `settings.json` file as pointed in #1478 issue?

**Comment by superandrew** [comment]
Unfortunately this is such an impredictable behaviour.. I tried removing the settings.json and the extension started for many vs code relaunches. But then I put the file in place again and it continue working. Usually my morning routine is open vs code, start the debugger, get the error, open developer tools, reload, then start the debugger and it works.

I can't tell if this changed but I can delete the file and see if in the next days whis will change. However this file doesn't seem to have anything wrong:

```
// Place your settings in this file to overwrite default and user settings.
{
  "javascript.validate.enable": false,
  "files.trimTrailingWhitespace": true,
  "eslint.enable": false,
  "editor.insertSpaces": true,
  "editor.tabSize": 4,
  "files.exclude": {
    "**/node_modules": true,
    "nls.*.json": true,
    "**/*.js": {
      "when": "$(basename).ts"
    },
    // "**/out": true,
    "**/*.js.map": true
  },
  "typescript.tsdk": "./node_modules/typescript/lib",
  "todo-tree.regex.regex": "(//|#|<!--|;|/?\\*|^)\\s*($TAGS)",
  "todo-tree.general.tags": [
    "@todo",
    "TODO",
    "FIXME",
    "UI"
  ],
  "todo-tree.filtering.ignoreGitSubmodules": true,
  "todo-tree.filtering.excludeGlobs": [
    "**/vendor/**",
    "**/node_modules/**",
    "**/dist/**",
    "**/bower_components/**",
    "**/build/**",
    "**/.vscode/**",
    "**/.github/**",
    "**/_output/**",
    "**/*.min.*",
    "**/*.map"
  ],
  "todo-tree.highlights.defaultHighlight": {
    "type": "whole-line",
    "background": "#FF0"
  }
}
```

**Comment by JiglioNero** [comment]
Hi @superandrew , could you please tell if you are still having this issue? If so, have you noticed any new patterns of its occurrence or do you have any assumptions about it?

**Comment by superandrew** [comment]
To be honest at the time I learned to live with it, then I started working with different technologies so I jumped into the RN projects after opening VS Code with another project (this seemed to be consistently a way to avoid the behavior). Now I am not able to recreate it anymore even if I close VS Code and open a RN project as first and immediately start debugging. 

Are you having the problem or you are asking it in order to close the issue? I'd say the issue is fixed for me but I'm not sure what solved the problem exactly.
